spec/factories/project.rb

```diff
@@ -14,23 +14,23 @@
       updated_by { FactoryBot.create(:user).uid }
       project_id { "" }
       status { "pending" }
-      storage_capacity { { size: { requested: 500 }, unit: { requested: "GB" } } }
-      storage_performance { { requested: "standard" } }
+      storage_capacity { { size: { requested: 500 }, unit: { requested: "GB" } }.with_indifferent_access }
+      storage_performance { { requested: "standard" }.with_indifferent_access }
       project_purpose { "research" }
       project_directory { "big-data" }
       schema_version { ::TigerdataSchema::SCHEMA_VERSION }
       approved_by { nil }
       approved_on { nil }
+      # TODO: remove submission from the project factory
```

Comment:
@hectorcorrea Submission is part of the metadata I'm unsure why you would remove it https://docs.google.com/document/d/1RDCowxE2ulfkh0GrqTln8Tr9t5XcWOZ2U--zRpiwe48/edit#heading=h.jnv7e4pqqw3u